tests/test_benchmark.py

```diff
@@ -330,6 +330,10 @@ def cleanup(self):
   benchmarkers += [
     EmscriptenBenchmarker(os.environ.get('EMBENCH_NAME') or 'v8', V8_ENGINE),
   ]
+if shared.NODE_JS and shared.NODE_JS in shared.JS_ENGINES:
```

Comment:
I don't think there's a need to check for `NODE_JS` existing - emscripten requires node.js to run, so it's guaranteed to be present - but I guess it's ok to leave it as maybe that will change in the far future.